src/pet.js

```diff
@@ -0,0 +1,82 @@
+const MAXIMUM_FITNESS = 10;
+const MINIMUM_HUNGER = 0;
+const FITNESS_THRESHOLD = 3;
+const HUNGER_THRESHOLD = 5;
+const AGE_LIMIT = 30;
+const HUNGER_LIMIT = 10;
+const FITNESS_LIMIT = 0;
+
+Pet.prototype = {
+    get isAlive() {
+        return (this.age < AGE_LIMIT) && (this.hunger < HUNGER_LIMIT) && (this.fitness > FITNESS_LIMIT);
```

Comment:
This could be made more readable by showing each condition on its own line:
```suggestion
        return (
            this.age < AGE_LIMIT &&
            this.hunger < HUNGER_LIMIT &&
            this.fitness > FITNESS_LIMIT
        );
```